Tests/Functional/Action/Platform/Service/OAuth2AccessTokenCreationActionTest.php

```diff
@@ -132,7 +132,11 @@ public function testWithInValidClientAssertion(): void
             ]
         );
 
-        $this->assertEquals(Response::HTTP_UNAUTHORIZED, $this->client->getResponse()->getStatusCode());
+        if (PHP_VERSION_ID < 70300) {
```

Comment:
this is due to underlying vendor (phpleague) behavior chnage in recent versions.

I keep it this way cause anyway we will have a 4xx, which is the most important.